qcengine/programs/cfour/runner.py

```diff
@@ -98,7 +98,7 @@ def build_input(
         opts.update(moldata["keywords"])
 
         # Handle calc type and quantum chemical method
-        mdcopts = muster_modelchem(input_model.model.method, input_model.driver.derivative_int())
+        mdcopts = muster_modelchem(input_model.model.method, input_model.driver)
```

Comment:
Nice. Switching from the integer to the text string is a good change to make on the way